.gitignore

```diff
@@ -0,0 +1,126 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+ # C extensions
+*.so
+
+ # Distribution / packaging
+.Python
```

Comment:
Are all these file generated today. Could we add only those files that needs to be ignored by git.

Reply:
I just copy that from [here](https://github.com/github/gitignore/blob/master/Python.gitignore), which hosts gitignore for various programming languages, and has been reviewed by nice people. Picking out those for this project doesn't worth it, personally speaking.